### ISSUE TITLE
make a default classifier from multiple datasets

### ISSUE THREAD

**Issue by marius10p** [MouseLand/suite2p#25]
This one should come as default with Suite2p.

**Comment by marius10p** [comment]
Trying to use Allen datasets for this, one for each CRE line, except EMX for which we will use several (it's a dense label).  Should consider adding neurofinder datasets, which are very different because they are short and more noisy.

**Comment by marius10p** [comment]
Allen datasets are noisy for most CRE lines and include weird pre-processing steps like pixel deconvolution. Will only use some of their EMX-CRE. Investigating the neurofinder datasets now. Also asking Mario for some of his 2x zoom data.

**Comment by carsen-stringer** [comment]
need to decide which stats to choose, currently: skew, compact, footprint

**Comment by marius10p** [comment]
Added normalized pixel count as statistic. Default classifier includes a single mesoscope recording with ~6,000 cells and ~6,000 non-cells at low zoom and 2:3 aspect ratio. The trained data was before we brought back connected regions, so some of the non-cells are scattered. Classifier histogram smoothing works as before except the grids are determined from the training data as quantiles rather than pre-defined. 

No other statistic helped performance on test data. We tried aspect_ratio, std, kurtosis, mean image correlation, and correlation with neuropil.
